### PR TITLE
Changes some default values from character creation and random characters

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Departmental Backpacks are now the default instead of grey backpacks.
Jumpsuits are now default for all characters.
## Why It's Good For The Game
jumpskirts should be an explicit choice

## Changelog
:cl:
image: Changed some default options in the character creator.
/:cl:

### CODE REVIEW

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (Mothblocks) [inline _maps/_basemap.dm:1]: erm

File: code/modules/client/preferences/clothing.dm [hunk 2]
```diff
@@ -80,6 +80,9 @@
 		PREF_SKIRT,
 	)
 
+/datum/preference/choiced/backpack/create_default_value()
+	return PREF_SUIT
```
Comment (Mothblocks) [inline code/modules/client/preferences/clothing.dm:84]: overriding backpack/create_default_value() again...?

**Comment by itseasytosee** [comment]
Sorry mothblocks its been a while.

**Review by MrMelbert — APPROVED** [review]

**Comment by itseasytosee** [comment]
These changes were both added incidentally in #84726. So this may be a partial revert if anything.

**Comment by munchyi** [comment]
retard

**Comment by vendingmachine24** [comment]
incel

**Comment by optimumtact** [comment]
> Jump skirts are comedic. They are frivolous, completely unrealistic, and with many of the sprites, overtly feminizing. By making them the default for women, we are essentially saying: "Women by default should be wearing clothes that extenuate their femininity, even at the cost of practicality." Many of the jumpskirt sprites are very puffy and frilly. This is despite the fact that jumpsuits are usually seen as agendered in workplace settings, and feminine in the fashion industry.

This is exceptionally poor reasoning for a change, and is breathless and overexcited and I was tempted to revert the PR entirely, but it's also a minor change so i'm not going to bother.

in the future make sure your reasons are balanced and sensible otherwise I'm just going to close your Pr's

**Comment by GremlinSeeker** [comment]
GBP farming I see

**Pull request merged**